pbsmetrics/prometheus/prometheus.go

```diff
@@ -28,7 +28,8 @@ type Metrics struct {
 	requestsWithoutCookie        *prometheus.CounterVec
 	storedImpressionsCacheResult *prometheus.CounterVec
 	storedRequestCacheResult     *prometheus.CounterVec
-	timeout_notifications        *prometheus.CounterVec
+	timeoutNotifications         *prometheus.CounterVec
+	tcfMetrics                   *prometheus.CounterVec
```

Comment:
Can we please call this `tcfVersion` or something along those lines so that we're consistent with the rest of the metric names?